holder.yml

```
@@ -107,7 +110,8 @@ paths:
        - oAuth2: []
        - zCap: []
       operationId: getPresentation
-      x-expectedCaller: "Holder Coordinator"
+      x-expectedCaller:
+        - Holder Coordinator
```

A workflow service could also conceivably use this endpoint.

```suggestion
        - Holder Coordinator
        - Workflow Service
```